library/standard_tables_capture.inc

```diff
@@ -243,6 +244,144 @@ function snomed_import($us_extension=FALSE) {
     return true;
 }
 
+function snomedRF2_import($us_extension=FALSE) {
```

Comment:
Remove the $us_extension variable since it's no longer used (now that the US extension is a complete package for RF2).